### PR TITLE
Create and improve cypress tests for the Modeling Exercises

### PR DESCRIPTION
**Pull request by rs-maker**
### Motivation and Context
 Improving end to end test coverage for Modeling Exercises.

### Description
 Added some new tests, fixed breaking bugs in the old ones.
Tests include: 
Create Modeling Exercise,
Create Example Solution
Create Example Submission
Edit Modeling Exercise
///////////
Exercise flow:
Releasing Exercise
Starting and Submitting Solution
Close Exercise Submissions
Assess Submission
Close Assessment
View assessment and handle complaints

### Steps for Testing
 Look at code and to run locally:
1. Pull branch
2. In cypress.json: set baseUrl to testserver of your choosing (ts2 or ts3 because you need admin access)
3. In cypress.env.json 
4. set "username" and "password" to the one used <a href="https://confluence.ase.in.tum.de/pages/viewpage.action?pageId=25252245"> here </a> except put "USERID" for the variable part. 
5. set "adminUsername" and "adminPassword" to an admin account you have access to
6. lastly set "isCi" to true
4. $ yarn run cypress open 
5. run ModelingExerciseSpec or ideally all specs to make sure everything works 
6. alternatively to run headlessly: $ yarn run cypress run --spec "src/test/cypress/integration/ModelingExercise.spec.ts"

You can also configure all of these options to test your locally hosted version of artemis, but i dont recommend it.

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -7,5 +7,6 @@
     "supportFile": "src/test/cypress/support/index.ts",
     "baseUrl": "http://localhost:8080",
     "video": false,
-    "screenshotOnRunFailure": false
+    "screenshotOnRunFailure": false,
+    "chromeWebSecurity": false
```
Comment (jhainzlschmid) [inline cypress.json:11]: Would be great if we could avoid using that flag

Comment (rs-maker) [reply cypress.json:11]: Fair point i forgot about that. ill try to fix it.

**Review by jhainzlschmid — COMMENTED** [review]
issue was fixed already my branch was just not up to date

**Review by jhainzlschmid — APPROVED** [review]
Tests work when executing them locally against the testserver as well as in the bamboo buildplan. The code will be refactored to use pageobjects in a followup PR after this PR was merged with [this cypress PR](https://github.com/ls1intum/Artemis/pull/3687)

**Pull request merged**